common/persistence/src/main/java/app/coronawarn/datadonation/common/persistence/service/SaltService.java

```diff
@@ -0,0 +1,29 @@
+package app.coronawarn.datadonation.common.persistence.service;
+
+import app.coronawarn.datadonation.common.persistence.repository.ppac.android.SaltRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SaltService {
+
+  @Autowired
+  private SaltRepository saltRepository;
+
+  Logger logger = LoggerFactory.getLogger(SaltService.class);
+
+  /**
+   * Deletes the salt provided.
+   *
+   * @param salt String salt
+   */
+  public void deleteSalt(String salt) {
+    try {
+      saltRepository.deleteSalt(salt);
+    } catch (RuntimeException saltNotFoundException) {
+      logger.error("Salt " + salt + " not found.");
```

Comment:
please add the original exception as well - what if it's another RuntimeException you currently don't know yet

Reply:
We can add that, but then I think it will be useful if we throw it further, not just log it, right ?